src/figuresClasses.ts

```diff
@@ -1,19 +1,66 @@
 export interface Figure {
-
+  shape: string;
```

Comment:
create enum for this and use it in code belove, because now it can be any string